src/components/evaluation/EvaluationTable.vue

```diff
@@ -468,7 +444,8 @@
                   </v-btn>
                   <v-btn
                     id="cancel-evaluation-edit-btn"
-                    class="ma-2 evaluation-form-btn"
+                    class="ma-2 evaluation-form-btn text-primary"
+                    color="bg-white"
```

Comment:
@mohamalnadi - does `color=bg-white` work as expected? I didn't think that prop accepted CSS style names but maybe it's smart like that.